static/src/javascripts/projects/common/modules/identity/consents.js

```diff
@@ -149,34 +149,38 @@ const confirmUnsubscriptionFromAll = (
             })
         );
 
+const unsubscribeFromAll = (csrfToken: string) =>
+    reqwest({
+        url: `/user/email-subscriptions`,
+        method: 'DELETE',
+        withCredentials: true,
+        headers: {
+            'Csrf-Token': csrfToken,
+        },
+    });
+
 const bindUnsubscribeFromAll = (buttonEl: HTMLButtonElement) => {
```

Comment:
Could we add a return type for this? (`Promise<void>`)? That one's on me. Forgot to add it the last time I was here!